data/siggraph-2018/10.1145-3197517.3201336.json

```diff
@@ -51,5 +51,57 @@
         "Misc. comments": "The project page also contains a windows executable if was not able to run on mac",
         "Software language": "C++"
+    },
+    {
+        "Variant name": "Ubuntu 18.04",
+        "Reviewer name": "Julie Digne <julie.digne@liris.cnrs.fr>",
+        "Is master variant (boolean)": false,
+        "Is variant deprecated (boolean)": true,
+        "Title": "Water surface wavelets",
+        "DOI": "10.1145/3197517.3201336",
+        "Year": 2018,
+        "ACM Keywords": [
+            "Physical simulation"
+        ],
+        "Topic {Rendering, Animation and Simulation, Geometry, Images, Virtual Reality, Fabrication}": "Animation",
+        "Co-authors from academia (boolean)": true,
+        "Co-authors from industry (boolean)": true,
+        "ACM Open Access (boolean)": true,
+        "PDF on the authors' webpage / institution (boolean)": true,
+        "PDF URL": "http://pub.ist.ac.at/group_wojtan/projects/2018_Jeschke_WaterSurfaceWavelets/WaterSurfaceWavelets.pdf",
+        "PDF on Arxiv or any openarchive initiatives (boolean)": false,
+        "Arxiv/OAI page URL": "",
+        "Project URL": "",
+        "Code available (boolean)": false,
+        "If code not available, pseudo-code available (boolean)": false,
+        "If pseudo-code, could the paper be trivially implemented? {0..4}": "",
+        "Code URL": "",
+        "Code URL2": "",
+        "MD5 sum (for archives)": "",
+        "git/hg/svn commit hash or revision number": "",
+        "MD5 sum (for archives) URL2": "",
+        "git/hg/svn commit hash or revision number URL2": "",
+        "Software Heritage permalink": "",
+        "Software type {Code, Binary, Partial Code}": "",
+        "Code License (if any)": "",
+        "Are the code authors explicit? (boolean)": false,
+        "Build/Configure mechanism": "",
+        "Dependencies": "",
+        "Does the software require paywall/proprietary software/material (boolean)?": false,
+        "Does the code need data (not examples) (boolean)": false,
+        "Nature of the data (pretrained model, LUT...)": "",
+        "License of the data": "",
+        "Able to perform a replicability test (boolean)": true,
+        "If not able to perform a test, was it due to missing hardware/software? (boolean)": false,
+        "Documentation score {0=NA,1,2,3}": 0,
+        "Dependencies score {0=NA, 1,2,3,4,5}": "0",
```

Comment:
should numbers (0) and not  strings ("0") 

The "" are misleading in the original json